sema4ai/vscode-client/src/common.ts

```diff
@@ -12,13 +12,23 @@ export type GetPackageTargetDirectoryMessages = {
     useWorkspaceFolderPrompt: string;
     useChildFolderPrompt: string;
     provideNamePrompt: string;
+    commandType: string;
 };
 
 export type WorkspacePackagesInfo = {
     agentPackages: LocalPackageMetadataInfo[];
     taskActionPackages: LocalPackageMetadataInfo[];
 };
 
+export type PackageTargetAndNameResult = {
+    targetDir: string | null;
+    name: string | null;
+};
+
+export const isActionPackageCommand = "isActionPackageCommand";
```

Comment:
As these are string constants they should be all uppercase (the way it is it seems like a function call).

i.e.: `IS_ACTION_PACKAGE_COMMAND` and the place they're used should use those literals directly for typing.

As a note, this would probably be better as an enum for typing, so, you'd type where it's used for the enum instead of just `string` -- note that inside an enum it doesn't need to be uppercase ;)

Reply:
Actually, you should probably just use `PackageType` here!